.travis.yml

```diff
@@ -14,7 +14,7 @@ install: true
 before_script:
 - sudo chown -R $USER:$GROUP $TRAVIS_BUILD_DIR
 script:
-- if [ "$TRAVIS_PULL_REQUEST" != "false" ] || [ "$TRAVIS_BRANCH" = "master" ]; then mvn clean install -q -U; fi
+- if [ "$TRAVIS_PULL_REQUEST" != "false" ] || [ "$TRAVIS_BRANCH" = "master" ]; then ./mvnw clean install -q -U; fi
```

Comment:
please revert to `mvn` as we dont want to use mvn wrapper for travis builds.

Reply:
I updated as requested.